test/e2e/testdata/ensure-e2e-deps/ensure-aws-cli.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
```

Comment:
lets move all the utils/scripts out of `testdata` directory, it should only be used for fixtures